mappings/net/minecraft/client/render/model/json/ModelTransformationData.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS dfj net/minecraft/client/render/model/json/ModelTransformationData
```

Comment:
Why not just... Transformation?

Reply:
Yeah that works, honestly slipped my mind..